admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ZipFileApplicationDetailsTest.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class ZipFileApplicationDetailsTest {
+    /**
+     * Tests the {@link ZipFileApplicationDetails#ZipFileApplicationDetails(String, String)} constructor,
+     * and the getters related to it
+     */
+    @Test
+    public void testConstructor() {
+        ZipFileApplicationDetails testZipFile = new ZipFileApplicationDetails("TestName", "0.0.0");
```

Comment:
Another general comment; test strings, especially test strings being asserted against, could be defined as constants instead. This makes the test less fragile to future changes and reduces duplication.